device/celestica/x86_64-cel_e1031-r0/platform_update_reboot_cause

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+REBOOT_USER=$(logname)
+REBOOT_TIME=$(date)
+declare -r REBOOT_CAUSE_FILE="/host/reboot-cause/reboot-cause.txt"
+
+echo "User issued 'reboot' with platform-specific command [User: ${REBOOT_USER}, Time: ${REBOOT_TIME}]" > ${REBOOT_CAUSE_FILE}
+sync
+/sbin/fstrim -av
```

Comment:
@qnos why do we need trim at this point?

Reply:
> @qnos why do we need trim at this point?

Cold reboot is a kind of force power cycle, add fstrim to try reducing the disk segment before force reboot. Commonly in non-test environment, cold reboot wouldn't be called frequently, so it wouldn't do harm to SSD health and lifecycle. Even in test environment, hundreds of reboot test would introduce obvious effect to SSD.

Reply:
@qnos reducing the disk segment means? how cold reboot affects SSD health/life?

Reply:
> 



> @qnos reducing the disk segment means? how cold reboot affects SSD health/life?

fstrim could be used for disk defragmentation, i meant using fstrim too frequently would affect SSD health. But in reboot case, that wouldn't be frequent.

Reply:
@qnos I think Prince's questions is on why we need do this fstrim here? Since we already have cronjob to do daily fstrim, it seems like unnecessary to do fstrim before power-cycle. Do you have any other concern to have this operation here? If not, I think we'd better remove it since it is not related to the power-cycle.